Generate/general/xslt/generateTestScript.xsl

```diff
@@ -205,25 +205,25 @@
             </resource>
         </fixture>
     </xsl:template>
-    
+
     <!-- Epand a nts:variables element; read all f:variable elements in the referenced file and further process them --> 
-    <xsl:template match="nts:variables[@href]" mode="expand" xmlns="http://hl7.org/fhir">
+    <xsl:template match="nts:include[@href]" mode="expand" xmlns="http://hl7.org/fhir">
         <xsl:param name="testscriptBase"/>
-        <xsl:variable name="loadedVariables" as="node()*">
+        <xsl:variable name="parts" as="node()*">
             <xsl:choose>
                 <xsl:when test="$testscriptBase">
-                    <xsl:copy-of select="document(@href, $testscriptBase)/f:variables/(element()|comment())"/>
+                    <xsl:copy-of select="document(@href, $testscriptBase)/nts:parts/(element()|comment())"/>
                 </xsl:when>
                 <xsl:otherwise>
-                    <xsl:copy-of select="document(@href)/f:variables/(element()|comment())"/>
+                    <xsl:copy-of select="document(@href)/nts:parts/(element()|comment())"/>
                 </xsl:otherwise>
             </xsl:choose>
         </xsl:variable>
         <!-- Note: don't pass testscriptBase here, because recursive includes are relative to the including file,
              not to testscriptBase -->
-        <xsl:apply-templates select="$loadedVariables" mode="expand"/>
+        <xsl:apply-templates select="$parts" mode="expand"/>

```

Comment:
Why apply-templates and not copy-of? If this is only meant to be an include mechanism, copy-of should suffice. Applying would be a different mechanism imho.

Reply:
Because a variables file may contain referencing elements itself, so it has to recursively apply the expansion step on the loaded elements.